src/objective-c/tests/CoreCronetEnd2EndTests/CoreCronetEnd2EndTests.m

```diff
@@ -236,7 +245,7 @@ - (void)testBadHostname {
 }
 
 - (void)testBinaryMetadata {
-  [self testIndividualCase:"binary_metadata"];
+  //[self testIndividualCase:"binary_metadata"];
```

Comment:
If the test is commented out, then why include the `testBinaryMetadata` method if the model of how it would be tested is not described?